content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -5,5 +5,261 @@ weight: 2
+
+7. **istioctl CLI Gaps:** Minor functional gaps may exist in the Istio CLI's output displays related to ambient mode. These will be addressed as the feature matures.
+
+In addition to this general caveats, there are also some specific caveats to be aware of when using Waypoint with certain protocols:
+
+- Waypoint only supports Ambient workloads. It does not support sidecar proxy workloads. 
+- HTTP: Waypoint does not support all HTTP features, such as HTTP/2 and chunked encoding.
+- gRPC: Waypoint does not support all gRPC features, such as HTTP/2 transport and protocol multiplexing.
+- WebSocket: Waypoint does not support WebSocket.
+
+Despite these caveats, Waypoint is a powerful tool for enabling L7 networking and services for Istio Ambient workloads. It is a good choice for users who are looking for a way to run microservices-based applications in Ambient mode.
+
+Here is a table summarizing the caveats of Waypoint:
+
+| Caveat | Description |
+| ------------- | ------------- |
+| Maturity | Waypoint is still under development |
+| Features | Waypoint only supports L7 load balancing and routing |
+|Integration | Waypoint is not yet fully integrated with the Istio control plane | Support | Waypoint only supports Ambient workloads | HTTP | Waypoint does not support all HTTP features | 
+gRPC | Waypoint does not support all gRPC features | Websocket | Waypoint does not support WebSocket |
+
+In addition to these caveats, it is also important to note that Waypoint is a new feature, and it is not yet as mature as Istio's sidecar proxy. As a result, users may experience some performance or stability issues when using Waypoint. However, the Ambient mesh team is actively working to address these issues, and they are committed to making Waypoint a production-ready feature.
+
+Overall, Waypoint is a powerful tool for enabling L7 networking and services for Istio Ambient workloads. However, users should be aware of the caveats and limitations listed above before deploying Waypoint in production.
+
+### Environment used for this guide
+
+For the examples in this guide, we used a deployment of Istio version `1.19.0` on a `kinD` cluster of version `0.20.0` running Kubernetes version `1.27.3`. However these should also work on any Kubernetes cluster at version `1.24.0` or later and Istio version `1.18.0` or later. It would be recommended to have a cluster with more than 1 worker node in order to fully exercise the examples described in this guide. Refer to the [Installation user](https://deploy-preview-13635--preliminary-istio.netlify.app/latest/docs/ops/ambient/usage/install/) guide or [Getting started guide](https://deploy-preview-13635--preliminary-istio.netlify.app/latest/docs/ops/ambient/getting-started/) on installing Istio in ambient mode on a Kubernetes cluster.
```

Comment:
Use relative pathes, for example /docs/ops/ambient/usage/install/

Reply:
I'd assume just pointing to env from L4 or get starte guide. no need to cover env here again.